### DISCUSSION TITLE
Linux Support

### DISCUSSION THREAD

**Discussion by iturdzonyou** [aleksilassila/reiverr#46] [Q&A] [answered]
Hi, will you be bringing this to Linux/Ubuntu? As of this time I only see it available via docker.

**Comment by aleksilassila** [comment]
I'm planning on adding more installation options shortly

**Reply by iturdzonyou** [reply]
Just did it outside of docker.

git clone https://github.com/aleksilassila/reiverr.git
cd reiverr
pnpm install
pnpm run build
pnpm run dev

Create .env file:
Fill in the blanks and change the base URLs to match your setup.

PUBLIC_RADARR_API_KEY=yourapikeyhere
PUBLIC_RADARR_BASE_URL=http://127.0.0.1:7878/

PUBLIC_SONARR_API_KEY=yourapikeyhere
PUBLIC_SONARR_BASE_URL=http://127.0.0.1:8989/

PUBLIC_JELLYFIN_API_KEY=yourapikeyhere
PUBLIC_JELLYFIN_URL=http://127.0.0.1:8096/

**Reply by aleksilassila** [reply, marked as answer]
That should work, you could also clone the project and remove the amd64 platform flag from the Dockerfile to build for your platform from source. If anyone knows a proper solution to supporting multiple platforms with a single image, let me know

Edit: #45
